### DISCUSSION TITLE
Problem running fit_textured_mesh.ipynb tutorial

### DISCUSSION THREAD

**Discussion by lynnfyffe** [facebookresearch/pytorch3d#1036] [Q&A] [answered]
Training loss goes to nan immediately when running texture prediction. Silhouette prediction succeeds with loss around 0.01. Predict textured (cell [38]) fails with loss nan. Is the notebook out of date?

Link: https://colab.research.google.com/github/facebookresearch/pytorch3d/blob/stable/docs/tutorials/fit_textured_mesh.ipynb

**Comment by bottler** [comment, marked as answer]
This is a known issue, #561. Adding `perspective_correct=False` to every `RasterizationSettings` fixes it.

**Reply by lynnfyffe** [reply]
Thank you! It works
